Modules/TRD/src/TrackletPerTriggerCheck.cxx

```diff
@@ -28,48 +29,29 @@
 
 using namespace std;
 using namespace o2::quality_control;
+using namespace o2::quality_control_modules::common;
 
 namespace o2::quality_control_modules::trd
 {
 
 void TrackletPerTriggerCheck::configure()
 {
   // ccdb setting
-  if (auto param = mCustomParameters.find("ccdbtimestamp"); param != mCustomParameters.end()) {
-    mTimeStamp = std::stol(mCustomParameters["ccdbtimestamp"]);
-    ILOG(Debug, Support) << "configure() : using ccdbtimestamp = " << mTimeStamp << ENDM;
-  } else {
-    mTimeStamp = o2::ccdb::getCurrentTimestamp();
-    ILOG(Debug, Support) << "configure() : using default timestam of now = " << mTimeStamp << ENDM;
-  }
+  auto mTimeStamp = getFromConfig<long>(mCustomParameters, "ccdbtimestamp", o2::ccdb::getCurrentTimestamp());
+  ILOG(Debug, Support) << "using timestamp = " << mTimeStamp << ENDM;
   auto& mgr = o2::ccdb::BasicCCDBManager::instance();
```

Comment:
what are you using the BasicCCDBManager for?

Reply:
I try to explain how I understood the concept of BasicCCDBManager. Although I do not fully understand.
1. instance() method of class BasicCCDBManager : public CCDBManagerInstance initialise the class with ccdb server. 
2. Then setTimestamp() method is used for setting timestamp manually if required to be given from config file otherwise current time stamp is used. It set timestamp cache for all queries (line no. 75 of BasicCCDBManager.h).
3. I see that BasicCCDBManager class itself by default set current timestamp (line no. 189 of BasicCCDBManager.h), so if we remove it from checker class it run without any problem. 
4. In lack of knowledge, I kept these line because I did not know its consequences.
 I will do how you suggest.

Reply:
I had in my mind that in background of workflow, It set the timestamp of QO object generated by checker class. And if a specific object is needed by any postprocessing in future, It can use that timestamp to query that QO object. Although this thought needs validation.

Reply:
Hi Deependra, I would simply remove the CCDB manager completely from your task, because to me it does not look like you are using it. In some cases people are requesting CCDB objects in their tasks and use the CCDB manager for it, but I don't think you need any calibration objects for your check so I would get rid of it to keep the code as clean as possible. Later we will need some CCDB objects for more advanced checks, but also there not the CCDB manager should be used anymore, but the more modern DPL CCDB fetcher. But we can talk about this when we get there :)

Reply:
Hi Ole, I removed that part.